integration-test/src/test/java/org/apache/iotdb/relational/it/rest/it/IoTDBRestServiceInsertValuesIT.java

```diff
@@ -51,7 +51,7 @@
 
 @RunWith(IoTDBTestRunner.class)
 @Category({LocalStandaloneIT.class, ClusterIT.class, RemoteIT.class})
```

Comment:
The IT for table should use `@Category({TableLocalStandaloneIT.class, TableClusterIT.class})`